### ISSUE TITLE
Docker Image for Builds

### ISSUE THREAD

**Issue by VikashKothary** [ankicommunity/docker-anki-sync-server#2]
We've now got a CI for https://github.com/ankicommunity/anki-sync-server. It needs a docker image for build process. I'm currently using this this image but it doesn't allow building the lastest verson of the `psutil` library.

Add this would fix that:
```
$ RUN apk add --update gcc libc-dev linux-headers && rm -rf /var/cache/apk/*
```
Source: https://github.com/giampaolo/psutil/issues/664.

Alternatively, we could have separate development and prod images, which would probably be better in the long term.

**Comment by VikashKothary** [comment]
Nope. The current docker image is for running the anki sync server and it runs fine. 

But for development we've got other libraries e.g. for testing, linting, building documentation, etc. Once of those libraries `psutil` doesn't work on alpine linux without the above dependencies. 

We could add the dependency to the existing docker image, however, that would increase the size of the image and would be pointless for 90% of users. This is why I was recommending creating a second docker image. 

I'll assign myself to the issue. Just wanted to run it past you first.

**Comment by kuklinistvan** [comment]
Ah I see, okay.

**Comment by fw-aaron** [comment]
@VikashKothary Would you mind submitting a pull request?

**Comment by VikashKothary** [comment]
@fw-aaron it would be much appreciated 👍🏽
